src/nftsLoop/nfts.ts

```
@@ -157,22 +168,116 @@ async function processNFTsForType(
+    mode,
+    nftMap,
+    newlyDiscoveredNfts,
+    collectionType
+  });
+}
+
+function createNftProcessingMap(
+  existing: Array<NFT | LabNFT>
+): Map<string, NftProcessingEntry> {
+  const nftMap = new Map<string, NftProcessingEntry>();
   existing.forEach((n) =>
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)